_posts/2022-06-05-May.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title:  "May - This Month in Minetest (04)"
+author: MisterE
+contributors: ["MisterE", "Rubenwardy"]
+description: >
+  This month, Minetest 5.5.1 is released, nodebox rendering gets a speedboost, we get a new mapping mod, and Minetest's first walkable vehicle. Exile and Nodecore get major improvements as well. 
+image: /static/blog/2022_May/izzyscastle.webp
+forum_topic: https://forum.minetest.net/viewtopic.php?f=18&t=28105
+tags:
+  - this_month
+---
+	First, we should celebrate a major milestone. For the first time in a long time, Minetest finally has fewer than 1000 open issues. A big thank you to the Development Team for speeding up the pace of development, and to Zughy for triaging the issues and closing the ones that could safely be closed.
+	
+  This month, Minetest 5.5.1 is released, nodebox rendering gets a speedboost, we get a new mapping mod and minetest's first walkable vehicle. Exile and Nodecore get major improvements as well. 
```

Comment:
Missing <!-- more --> after this paragraph

Reply:
explain?

Reply:
looks like GitHub swallowed that, I wrote `Missing <!-- more --> after this paragraph`

Reply:
so, a link to more info?